### PR TITLE
Jdr mpu fix

### PR DESCRIPTION
**Pull request by Joshua-David**
## What does this change?
Sets a fixed height for the ad container as a hotfix for sticky mpu's. Design to talk about alignment

### CODE REVIEW

File: static/src/stylesheets/module/content-garnett/_types.scss [hunk 1]
```diff
@@ -709,6 +709,10 @@ $quote-mark: 35px;
             }
         }
     }
+    // A fixed height is being added as a hotfix until final design decisions made for ad placement, important overrides current JS
```
Comment (sndrs) [inline static/src/stylesheets/module/content-garnett/_types.scss]: overrides the JS?

Comment (Joshua-David) [reply static/src/stylesheets/module/content-garnett/_types.scss]: There's a js file that applies sticky behaviour to the MPU and generates a height, this css value just gives a fixed height to all the ad slots as a hotfix (so supersedes that js value), plan is to have further discussion next week on placement etc. and then the JS file can be looked at

Comment (sndrs) [reply static/src/stylesheets/module/content-garnett/_types.scss]: why not disable the height being set in the JS? it doesn't feel very safe to have a set of things all trying to set a height on something and have them override each other?

**Comment by Joshua-David** [comment]
@sndrs I've commented out the height value in the JS, keeping it there as a comment because this is all just a hotfix until we decide where we want ads to sit

**Review by sndrs — APPROVED** [review]

**Comment by JonNorman** [comment]
Sorry, only just saw this; would it make more sense to just comment out wherever `sticky-mpu` is being invoked rather than running/including the file and then commenting out the final effect?

**Comment by Joshua-David** [comment]
But we still want the sticky behaviour, it's just the height of the container we are hard coding in. This has already been approved/merged but we can come up with a better solution in a new PR? Happy to follow your lead.

**Comment by JonNorman** [comment]
Ah sorry I deleted my comment as I saw that there was other stuff going on. All good! 👍

**Pull request merged**